quantum/rgblight.c

```diff
@@ -564,41 +573,77 @@ void rgblight_timer_toggle(void) {
 
 void rgblight_show_solid_color(uint8_t r, uint8_t g, uint8_t b) {
   rgblight_enable();
-  rgblight_mode(1);
+  rgblight_mode(RGBLIGHT_MODE_STATIC_LIGHT);
   rgblight_setrgb(r, g, b);
 }
 
 void rgblight_task(void) {
   if (rgblight_timer_enabled) {
     // mode = 1, static light, do nothing here
-    if (rgblight_config.mode >= 2 && rgblight_config.mode <= 5) {
+    if ( 1 == 0 ) { //dummy
+    }
+#ifdef RGBLIGHT_EFFECT_BREATHING
+    else if (rgblight_config.mode >= RGBLIGHT_MODE_BREATHING  &&
+        rgblight_config.mode <= RGBLIGHT_MODE_BREATHING_end) {
       // mode = 2 to 5, breathing mode
```

Comment:
These numbers e.g. `2 to 5` are no longer relevant, so the comments should probably be removed/updated (similarly for the other hardcoded mode numbers in corresponding comments below)